docs/source/membership/membership.md

```diff
@@ -1,341 +1,194 @@
+
+![MSP1a](./membership.msp.diagram.png)
+
+*Identities are similar to your credit cards that are used to prove you can pay. The MSP is similar to the list of accepted credit cards.*
+
+Consider a consortium of banks that operate a blockchain network. Each bank operates peer and ordering nodes, and the peers endorse transactions submitted to the network. However, each bank would also have departments and account holders. The account holders would belong to each organization, but would not run nodes on the network. They would only interact with the system from their mobile or web application. So how does the network recognize and differentiate these identities? A CA was used to create the identities, but like the card example, those identities cant just be issued, they need to be recognized by the network. MSP's are used to define the organizations that are trusted by the network members. MSPs are also the mechanism that provide members with a set of roles and permissions within the network.
+
+Finally, consider if you want to join an _existing_ network, you need a way to turn your identity into something that is recognized by the network. The MSP is the mechanism that enables you to participate on a permissioned blockchain network. To transact on a Fabric network a member needs to:
+
+- Have an identity issued by a CA that is trusted by the network.
+- Become a member of an _organization_ that is recognized and approved by the network members. The MSP is how the identity is linked to the membership of an organization. Membership is achieved by adding the member's public key (also known as certificate, signing cert, or signcert) to the organization’s MSP.
+
+## What is an MSP?
+
+Despite its name, the Membership Service Provider does not actually provide anything. Rather, the implementation of the MSP requirement is a folder that contains a list of related identities and is used to store users's certificates or in the case of channel configuration, a list of organizations. Whereas Certificate Authorities generate the certificates that represent identities, the MSP contains a list of permissioned identities.
+
+The MSP identifies which Root CAs and Intermediate CAs are accepted to define the members of a trust domain by listing the identities of their members, or by identifying which CAs are authorized to issue valid identities for their members.
+
+But the power of an MSP goes beyond simply listing who is a network participant or member of a channel. It is the MSP that turns an identity into a **role** by identifying specific privileges an actor has on a node or channel. Note that when a user is registered with a Fabric CA, a role of admin, peer, client, orderer, or member must be associated with the user.
```

Comment:
Last sentence --- might make sense to say that there are both roles and Fabric Roles. Something like:

This is related to the "role" explicitly defined by the Fabric CA when an identity is registered. Identities registered with the "peer" role will, naturally, be given to a peer. Similarly, identities registered with the "admin" role will be given to organization admins. All identities must be registered with a role, choosing between "admin", "peer", "client", "orderer", or "member". The structure of the channel MSP will then determine the role these identities have on the channel (for example, defining than an organization, and therefore its admin, must approve of changes to the channel configuration).

Reply:
I liked the clarification you suggested here and added it but left out the sentence about the channel MSP since we've not introduced that concept yet in the doc.